staging/src/k8s.io/apiserver/pkg/storage/tests/cacher_test.go

```diff
@@ -224,6 +224,48 @@ func TestGetToList(t *testing.T) {
 	}
 }
 
+func TestListWithLimitAndRV0(t *testing.T) {
+	cacher, _ := newTestCacher(nil, 0)
+	defer cacher.Stop()
+
+	pred := storage.SelectionPredicate{
+		Limit: 500,
+	}
+	result := &example.PodList{}
+
+	// While List with Limit should be served from underlying storage,
+	// if RV=0 is set, it should be served from cache.
+	// We pass nil underlying storage, so if the request would be
+	// pas through, it would panic.
+	defer func() {
+		if err := recover(); err == nil {
+			t.Errorf("List with Limit and RV=0 should be served from cache")
+		}
+	}()
+	cacher.List(context.TODO(), "pods/ns", "50", pred, result)
```

Comment:
this is confusing... the test title says RV0 but we're passing "50" here, right?